### PR TITLE
docs(self-managed): bitnami images can be pulled from Camunda's Docker Registry

### PR DESCRIPTION
**Pull request by clementnero**
## Description

Related to https://github.com/camunda/product-hub/issues/1075 and https://github.com/camunda/team-infrastructure/issues/689.

Bitnami (third-party) images can now be pulled from  Camunda's Docker Registry (pull-through cache) as well.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/setup/guides/air-gapped-installation.md [hunk 1]
```diff
@@ -39,13 +39,16 @@ The following images must be available in your air-gapped environment:
 
 We currently have a script in the [camunda-helm-respository](https://github.com/camunda/camunda-platform-helm/blob/c6a6e0c327f2acb8746802fbe03b3774b8284de3/scripts/download-chart-docker-images.sh) that will assist in pulling and saving Docker images.
 
-Please note that all the required Docker images, available on DockerHub's Camunda organization, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>`
+Please note that all the required Docker images, available on DockerHub's Camunda and Bitnami organizations, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>` and `registry.camunda.cloud/bitnami/<image>`
```
Comment (maxdanilov) [inline docs/self-managed/setup/guides/air-gapped-installation.md:42]: I've always missed a header before this paragraph, for the ability to make a deeplink to the segment on this page allowing to the Harbor as a mirror option, wdyt?

Something like:
```
## Accessing Camunda images from Camunda Docker Registry
```

**Review by maxdanilov — COMMENTED** [review]
LGTM, just offering a small UX improvement to add an additional header for deeplinking possibility.

File: docs/self-managed/setup/guides/air-gapped-installation.md (hunk 1 above)
Comment (clementnero) [reply docs/self-managed/setup/guides/air-gapped-installation.md:42]: I like it, thank you!

**Review by maxdanilov — DISMISSED** [review]

File: docs/self-managed/setup/guides/air-gapped-installation.md [hunk 2]
```diff
@@ -39,13 +39,18 @@ The following images must be available in your air-gapped environment:
 
 We currently have a script in the [camunda-helm-respository](https://github.com/camunda/camunda-platform-helm/blob/c6a6e0c327f2acb8746802fbe03b3774b8284de3/scripts/download-chart-docker-images.sh) that will assist in pulling and saving Docker images.
 
-Please note that all the required Docker images, available on DockerHub's Camunda organization, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>`
+## Accessing Camunda images from Camunda Docker Registry
 
-For example, the Docker image of Zeebe can be pulled via DockerHub or via the Camunda's Docker Registry:
+Please note that all the required Docker images, available on DockerHub's Camunda and Bitnami organizations, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>` and `registry.camunda.cloud/bitnami/<image>`
```
Comment (conceptualshark) [inline docs/self-managed/setup/guides/air-gapped-installation.md:44]:
```suggestion
All required Docker images, available on DockerHub's Camunda and Bitnami organizations, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>` and `registry.camunda.cloud/bitnami/<image>`
```

**Review by conceptualshark — DISMISSED** [review]
Wording suggestion, but non-blocking to merge - otherwise good from me. 👍

**Pull request merged**